index.js

```diff
@@ -14,54 +16,111 @@ module.exports.pitch = function(remainingRequest) {
 			regExp: query.regExp
 		};
 		var chunkName = loaderUtils.interpolateName(this, query.name, options);
-		var chunkNameParam = ", " + JSON.stringify(chunkName);		
+		var chunkNameParam = ", " + JSON.stringify(chunkName);
 	} else {
 		var chunkNameParam = '';
 	}
 	var result;
 	if(query.lazy) {
 		result = [
-			"module.exports = function(cb) {\n",
-			"	require.ensure([], function(require) {\n",
-			"		cb(require(", loaderUtils.stringifyRequest(this, "!!" + remainingRequest), "));\n",
+			"module.exports = function(successCallback, errorCallback) {\n",
```

Comment:
`function (successCallback, errorCallback)` => `function(cb, err)`

Reply:
not sure quite what you're suggesting I change here?